test/backend/inference_backend.py

```diff
@@ -77,13 +77,16 @@ def get_test_models():
         # ==ARCH== cpu
 
         # ==OP== Abs
+        # ==MIN== 13
```

Comment:
A quick read seems to indicate that we do support abs 6, and in fact we may not support 13's float

https://onnx.ai/onnx/operators/text_diff_Abs_6_13.html

Reply:
Sounds good; I would then be in favor of putting it to 7 if you believe that is appropriate